icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -286,14 +419,34 @@ private static LongNameHandler forCompoundUnit(
             // Some "one" pattern may not contain "{0}". For example in "ar" or "ne" locale.
             String secondaryCompiled = SimpleFormatterImpl
                     .compileToStringMinMaxArguments(secondaryFormat, sb, 0, 1);
-            String secondaryString = SimpleFormatterImpl.getTextWithNoArguments(secondaryCompiled)
-                    .trim();
+            String secondaryFormatString = SimpleFormatterImpl.getTextWithNoArguments(secondaryCompiled);
+
+            // TODO: do not use regular expression
```

Comment:
```suggestion
            // TODO(icu-units#27): do not use regular expression
```

Just a suggestion to help distinguish "our" TODOs from any old TODOs that might be lying around from before the units work.

Reply:
okay